loopy/target/pyopencl.py

```diff
@@ -799,7 +799,9 @@ def get_kernel_call(self, codegen_state, name, gsize, lsize, extra_args):
             Comment("{{{ enqueue %s" % name),
             Line(),
             Assign("_lpy_knl", "_lpy_cl_kernels."+name),
-            Assert("_lpy_knl.num_args == %d" % cl_arg_count),
+            Assert(f"_lpy_knl.num_args == {cl_arg_count}, f'Kernel \"{name}\" "
+                   f"setup arg count ({cl_arg_count}) does not match actual arg "
+                   "count ({_lpy_knl.num_args}).'"),
```

Comment:
Terminology: I don't know what "setup arg count" and "actual arg count" are. Please try to be precise with words.

Reply:
What do you think of [fba3427](https://github.com/inducer/loopy/pull/604/commits/fba34270b391b396ebf8b2d6c1518a07ebbd94b6)?